src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -73,8 +56,9 @@ public void equals_True_CI_101() {
     "DCC, 700",
     "DCCC, 800"
   })
-  void convertFromStringReturnsCorrectInt(String input, Integer expected) {
-    int actual = RomanNumeral.convertFromString(input);
-    assertTrue(expected.equals(actual));
+  void convertFromStringReturnsCorrectInt(String input, int expected) {
+    Integer expectedInteger = Integer.valueOf(expected);
+    Integer actual = Integer.valueOf(RomanNumeral.convertFromString(input));
+    assertTrue(expectedInteger.equals(actual));
```

Comment:
This isn't testing our equals() method, which is what this PR is supposed to be for.

You could change the test to this:
```
assertEquals(new RomanNumeral(input), new RomanNumeral(expected));
```

Reply:
I'm unclear on how this doesn't test equals() along with the confusion about the comments above. Either way I changed them. :)